src/uct/cuda/cuda_copy/cuda_copy_md.c

```
@@ -553,8 +554,7 @@ static void uct_cuda_copy_sync_memops(uct_cuda_copy_md_t *md,
 
     if (is_vmm) {
         ucs_fatal("failed to set sync_memops on CUDA VMM without "
-                  "cuCtxSetFlags() (address=%p)",
-                  address);
+                  "cuCtxSetFlags() (address=%p)", address);
```

Thinking of it again it should be a warning, since failure in cuPointerSetAttribute() call is also a warning

---

so when `is_vmm == 1` you want to call `cuPointerSetAttribute` and let it fail right?

---

moved to ucs_warn

---

hmm right, actually we can return from the function after ucs_warn, and not call cuPointerSetAttribute at all